electron/main.ts

```diff
@@ -1469,7 +1471,19 @@ ipcMain.handle(
 ipcMain.handle('getShellPath', async (event, path) => {
   return (await execAsync(`echo "${path}"`)).stdout.trim()
 })
-ipcMain.handle('getRealPath', (event, path) => realpathSync(path))
+ipcMain.handle('getRealPath', (event, path) => {
+  let resolvedPath = path
+  try {
+    resolvedPath = realpathSync(path)
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  } catch (err: any) {
```

Comment:
instead of `any` try using `Record<string, string>`

Reply:
Getting this if type is other than any or unknown

`error TS1196: Catch clause variable type annotation must be 'any' or 'unknown' if specified.`